README.md

```diff
@@ -2,14 +2,16 @@
 
 [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/dotnet/interactive/master?urlpath=lab) [![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/interactive/interactive-ci?branchName=master)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=744&branchName=master)
 
-.NET interactive provides data scientists and developers a way to explore data, experiment with code, and try new ideas effortlessly. Use .NET Interactive to build .NET Jupyter notebooks or custom interactive coding experiences.
+.NET Interactive gives developers, data scientists, makers, and learners tools to write code and see the results immediately. You can explore data, experiment with code, and try new ideas. Use .NET Interactive to build .NET Jupyter notebooks or custom interactive coding experiences including bots and custom REPLs.
 
-### Jupyter Notebooks with .NET
+## Jupyter Notebooks with .NET <img src ="https://upload.wikimedia.org/wikipedia/commons/thumb/3/38/Jupyter_logo.svg/207px-Jupyter_logo.svg.png" height="1.5em" alt="Project Jupyter" >
+
+[Project Jupyter](https://jupyter.org/) is a popular platform for creating interactive notebooks that can be used for data science, documentation, DevOps, and much more.
 
 <img src = "https://user-images.githubusercontent.com/547415/78056370-ddd0cc00-7339-11ea-9379-c40f8b5c1ae5.png" width = "70%">
 <img src="https://user-images.githubusercontent.com/2546640/67912370-1b99b080-fb60-11e9-9839-0058d02488cf.png" width="62%">
 
-# Jupyter Notebooks with .NET Core | Preview 2 <img src ="https://upload.wikimedia.org/wikipedia/commons/thumb/3/38/Jupyter_logo.svg/207px-Jupyter_logo.svg.png" width="38px" alt="dotnet bot in space" align ="right">
+# Jupyter Notebooks with .NET Core | Preview 2 
```

Comment:
Should we drop Jupyter and just say notebooks? Below we mention how to get started with Jupyter and nteract